### PR TITLE
[docs]: add gelato vrf

### PR DESCRIPTION
**Pull request by anirudhmakhana**
**What changed? Why?**
Added Gelato VRF to oracles section of the tools docs.

### CODE REVIEW

**Review by wbnns — CHANGES_REQUESTED** [review]
@anirudhmakhana 

Hi there! Thanks for this PR! Can you please omit the changes to `data.json` from this PR?

**Comment by cb-heimdall** [comment]
Review Error for wbnns @ 2024-05-14 22:52:27 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

File: apps/base-docs/docs/tools/oracles.md [hunk 1]
```diff
@@ -75,6 +77,17 @@ See [this guide](https://docs.diadata.org/introduction/intro-to-dia-oracles/requ
...
+
+Gelato VRF (Verifiable Random Function) provides a unique system offering trustable randomness on Base.
+
+See this guide to learn how to get started with [Gelato VRF](https://docs.gelato.network/web3-services/vrf/quick-start).
+
+#### Supported Networks
+
+- Base Mainnet
+- Base Sepolia (Testnet)
+
```
Comment (wbnns) [inline apps/base-docs/docs/tools/oracles.md:90]: @anirudhmakhana 

Can you also please add a divider below the Gelato entry, to match the others?

Comment (anirudhmakhana) [reply apps/base-docs/docs/tools/oracles.md:90]: Thanks @wbnns i have made the requested changes!

**Comment by zencephalon** [comment]
@anirudhmakhana Instead of deleting `data.json` could U just not introduce any changes to it? Thank U!

**Review by zencephalon — DISMISSED** [review]
Please leave `data.json` alone, it's currently being deleted

**Comment by cb-heimdall** [comment]
Review Error for zencephalon @ 2024-05-23 16:51:38 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by wbnns — APPROVED** [review]
Dropped the artifact

**Comment by cb-heimdall** [comment]
Review Error for wbnns @ 2024-05-24 10:51:43 UTC
User cannot review their own commit

**Review by rcbgr — APPROVED** [review]

**Pull request merged**